tests/scripts/gather-code-coverage.sh

```diff
@@ -7,6 +7,7 @@
 
 # First parameter is the name of the generated info file
 INFO_FILE=${1:-coverage.info}
+SRC_DIR="~/rpmbuild/BUILD/bluechi-1.2.0"
```

Comment:
We need to replace the hardcoded version here `bluechi-1.2.0` with a parameter. Could you have look at this? @ArtiomDivak

Reply:
Changed to dynamic version of bluechi